src/app/CommandHandler.cpp

```diff
@@ -436,7 +436,17 @@ TLV::TLVWriter * CommandHandler::GetCommandDataIBTLVWriter()
 
 FabricIndex CommandHandler::GetAccessingFabricIndex() const
 {
-    return mpExchangeCtx->GetSessionHandle()->AsSecureSession()->GetFabricIndex();
+    FabricIndex fabric = kUndefinedFabricIndex;
+    if (mpExchangeCtx->GetSessionHandle()->IsGroupSession())
+    {
+        fabric = mpExchangeCtx->GetSessionHandle()->AsGroupSession()->GetFabricIndex();
+    }
+    else
+    {
+        fabric = mpExchangeCtx->GetSessionHandle()->AsSecureSession()->GetFabricIndex();
+    }
```

Comment:
This logic should basically live on SessionHandle instead of copy/pasting it various places, but a followup is OK for that.

Reply:
It's another temp fix just to get groups back on tracks. You had already created an issue for it.
https://github.com/project-chip/connectedhomeip/issues/13396